app/model/commands/CreateWorkflowAtomCommand.scala

```diff
@@ -0,0 +1,89 @@
+package model.commands
+
+import java.util.Date
+import java.util.UUID._
+
+import com.gu.atom.data.IDConflictError
+import com.gu.contentatom.thrift.atom.media.{Category => ThriftCategory, MediaAtom => ThriftMediaAtom, Metadata => ThriftMetadata}
+import com.gu.contentatom.thrift.{Atom => ThriftAtom, _}
+import com.gu.media.logging.Logging
+import com.gu.pandomainauth.model.{User => PandaUser}
+import data.DataStores
+import model.commands.CommandExceptions._
+import model.{ChangeRecord, MediaAtom, WorkflowMediaAtom}
+
+import scala.util.{Failure, Success}
+
+case class CreateWorkflowAtomCommand(workflowMediaAtom: WorkflowMediaAtom, override val stores: DataStores, user: PandaUser)
+
+  extends Command with Logging {
+
+  type T = MediaAtom
+
+  def process(): MediaAtom = {
+    val atomId = randomUUID().toString
```

Comment:
As I just discussed with @LATaylor-guardian about this,  please find below the 4 reasons:

- Content API consistency: currently the atom with `id`:  `acb90e30-85a7-4d80-8dfa-a4d1f3fae642` is accessible on `https://content.guardianapis.com/atom/explainer/acb90e30-85a7-4d80-8dfa-a4d1f3fae642` on content API. This does not follow the rule that any item should be accessed by its `id`. Worse than than that, if you use the `atoms` endpoint of the `API` you can't find how to navigate to this `item` in particular, you have to know how to construct the `path` yourself, which is a leaky abstraction.

- Frontend consistency: Without registering the path to `path-manager`, nothing prevent a page to be registered with the same `id` than an atom. This is very unlikely but as some atoms may be rendered alone (without an article), it is easier to ensure it will never happened.  

- Nice urls: Some atoms could be rendered alone by frontend, and will use their `id` as the path (with a potential prefix such as `embed`). Having nice urls exactly like on article have several benefits. 

- Ease of debugging: Having the `type` and a summary of the content of the atom is very useful when debugging. it is easier to remember.

Note there is no plan of migrating existing atoms, only new atoms created when the code will be ready we have a `better` id.